tools/flashPXParameters.py

```diff
@@ -32,10 +32,78 @@
 # Port settings
 port = ''
 if platform.system() == 'Linux':
-	port = '/dev/ttyACM0'
+	port = '/dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00'
 elif platform.system() == 'Darwin':
 	port = '/dev/tty.usbmodem1'
 
+
+if not os.path.exists(port):
+    raise Exception("port does not exist: %s" % port)
+
+master = mavutil.mavlink_connection(port)
+master.wait_heartbeat()
+
+# Reset parameters to firmware defaults
+print "Resetting parameters to defaults.",
+
+master.mav.command_long_send(
+1, # target system
```

Comment:
[you should indent 'continuation lines'](https://www.python.org/dev/peps/pep-0008/#indentation), from what I understand you can use as many spaces as you want, whatever looks best.